spec/features/admin_creates_new_book_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+feature 'Admin creates a new book' do
+  scenario 'successfully' do
+    admin = create(:user, :admin)
+    visit root_path(as: admin)
+
+    create_book('The Fellowship of the Ring', 'J. R. R. Tolkien' )
+
+    expect(page).to have_content(t('books.index.header'))
+    expect(page).to have_content(t('books.create.flash_success'))
+    expect(page).to have_css('.books', text: 'The Fellowship of the Ring')
+    expect(page).to have_css('.books', text: 'J. R. R. Tolkien')
+  end
+
+  scenario "unsuccessfully - required field is missing" do
+    admin = create(:user, :admin)
+    visit root_path(as: admin)
+
+    create_book('The Two Towers', '')
+
+    expect(page).to have_content(t('books.new.header'))
+    expect(page).to have_content(t('books.validations.required'))
+  end
+
+  def create_book(title, author)
+    click_on t('books.index.add_new_book_link')
+    fill_in t('simple_form.labels.title'), with: title
+    fill_in t('simple_form.labels.author'), with: author
+    click_button t('books.create.button')
+  end
+end
+
+feature 'Patron (non admin) cannot create a book' do
```

Comment:
Usually there should be just one feature per feature spec. I was going to suggest that you split these up or reword it, but I think this reveals that maybe the link should never be on the index page and instead there should be a separate admin dashboard or something where you can create books. That may be worth doing. Then you don't need to test for the patron since you'll have `require_admin` as a `before_action`